libs/utils/Bitfield.lua

```diff
@@ -120,13 +120,16 @@ function Bitfield:__div(other)
 	end
 end
 
-function Bitfield:toArray(filter)
+function Bitfield:toArray(filter, sort)
 	local arr = {}
 	for k, v in pairs(checkType('table', filter)) do
 		if self:hasValue(v) then
 			insert(arr, k)
 		end
 	end
+	if sort and typing.checkType('table', sort) then
```

Comment:
`table` is the wrong type here. It must be a `function`.

Reply:
copy-paste mistake, thanks

Reply:
`sort` cannot be false, should be `if sort ~= nil` instead if I'm not mistaken

Reply:
type checking covers any other case

Reply:
If this is required, alright.